lib/HTFeed/JobMetrics.pm

```diff
@@ -0,0 +1,310 @@
+package HTFeed::JobMetrics;
```

Comment:
@aelkiss if you want to start looking before I'm 100% done with the PR you could start with this class.